Cargo.toml

```diff
@@ -17,4 +17,4 @@ maintenance = { status = "actively-developed" }
 [dependencies]
 serde_json = "1.0.38"
 clap = "2.31.2"
-exit = { git = "https://github.com/JoshMcguigan/exit" }
+exit = { git = "https://github.com/JoshMcguigan/exit", rev = "216e9b07" }
```

Comment:
Rather than push `exit` to crates.io in it's current state, I'd prefer to just lock the revision within your crate if it's okay with you. This protects your crate from breaking changes. 

I just want to be sensitive to taking names on crates.io for something like this `exit` crate which may or may not get long term support. If it grows into a robust solution to this problem, I'd be okay with publishing it, but for now I'd like to just give it some time to see if people take interest.

Reply:
Totally fine with me.